pkg/cidata/cidata.TEMPLATE.d/boot/30-install-packages.sh

```diff
@@ -67,6 +67,11 @@ elif command -v dnf >/dev/null 2>&1; then
 			dnf_install_flags="${dnf_install_flags} --repo ol8_baseos_latest --repo ol8_codeready_builder"
 		elif grep -q "release 8" /etc/system-release; then
 			dnf_install_flags="${dnf_install_flags} --enablerepo powertools"
+		else
+			# Workaround for disappeared sshfs in Base repos
+			# shellcheck disable=SC2086
+			dnf install ${dnf_install_flags} epel-release
+			dnf_install_flags="${dnf_install_flags} --enablerepo epel"
```

Comment:
Unneeded on Fedora and its non-EL derivatives (how to detect?)

Reply:
The systemd os-release isn't very helpful:

```
ID="centos"
ID_LIKE="rhel fedora"
```

i.e. it has _both_ of rhel (epel) and fedora

Reply:
But probably looking for "rhel" in either ID or ID_LIKE field of /etc/os-release, should be enough for "is EL" (needs EPEL)

Reply:
@AkihiroSuda @afbjorklund Sorry I'm absent for some days.

I didn't realize that this need to be restricted to rhel releases. So i pushed a new commit on the branch. I tested `grep 'ID_LIKE=".*rhel.*"' /etc/os-releases` on theses containers and this should work fine.

Is there any other problem? It looks good to me now.

Reply:
I think enabling EPEL should be up to the system administrator, and not modified by lima.

So if you enable it just for installing "sshfs", it should be disabled again after the installation.

Reply:
@afbjorklund good point. So what about this?

- install epel-release package and thus has it enabled by default
- disable it like `dnf config-manager --set-disabled epel-release` or something similar
- install sshfs with epel-release temporarily enabled with `--enablerepo=epel-release`

Reply:
Yes, this was my idea. Somewhat similar to what was already being done for CentOS 7.

```diff
--- a/pkg/cidata/cidata.TEMPLATE.d/boot/30-install-packages.sh
+++ b/pkg/cidata/cidata.TEMPLATE.d/boot/30-install-packages.sh
@@ -67,6 +67,10 @@ elif command -v dnf >/dev/null 2>&1; then
 			dnf_install_flags="${dnf_install_flags} --repo ol8_baseos_latest --repo ol8_codeready_builder"
 		elif grep -q "release 8" /etc/system-release; then
 			dnf_install_flags="${dnf_install_flags} --enablerepo powertools"
+		elif grep -q "release 9" /etc/system-release; then
+			dnf install ${dnf_install_flags} epel-release
+			dnf config-manager --disable epel >/dev/null 2>&1
+			dnf_install_flags="${dnf_install_flags} --enablerepo epel"
 		fi
 		# shellcheck disable=SC2086
 		dnf install ${dnf_install_flags} ${pkgs}

```

Reply:
https://github.com/lima-vm/lima/pull/901/commits/5fff4534b5eed0af7322ade6ae36469ee98326d0